src/views/virtualmachines/tree/utils/utils.tsx

```diff
@@ -129,7 +129,11 @@ const createProjectTreeItem = (
     treeViewDataMap,
   );
 
-  const projectChildren = [...projectFolders, ...(projectMap[project]?.ungrouped || [])];
+  const sortProjectFolders = projectFolders.sort((folderA, folderB) =>
+    folderA.id.localeCompare(folderB.id),
+  );
```

Comment:
I need to sort folders as sometimes they swap orders when I drag the VM from a folder to another